api/src/pcapi/core/auth/api.py

```diff
@@ -94,7 +92,7 @@ def extract_roles_from_google_workspace_groups(api_response: dict) -> set[str]:
             #      instead of backoffice-staging-admin) or
             #   2. defined env does not match the current env
             env = match.group("env")
-            if env and env.lower() != settings.ENV:
+            if not env or env.lower() != settings.ENV:
```

Comment:
si `match` n'est pas `None`, alors selon ma compréhension, `not env` ne devrait pas être possible puisque tu as enlevé le `?` de l'expression régulière

Reply:
Bien vu